Makefile

```diff
@@ -75,11 +79,12 @@ build: ## Build the project (install vendor, migrations, reindex, build assets,
 	@$(MAKE) -s vendor
 	@$(MAKE) -s migrations
 	@$(MAKE) -s reindex
-	ifeq ($(SYMFONY_ENV), prod)
-		$(MAKE) -s assets-prod
-	else
-		$(MAKE) -s assets
-	endif
+ifeq ($(SYMFONY_ENV), prod)
```

Comment:
Also not directly related to the PR, but the indent was wrong and Makefile did not work without this fix :)